### PR TITLE
Newsfeed

### PR DESCRIPTION
**Pull request by Escanord**
complete database connection to write & read posts information in newsfeed

### CODE REVIEW

File: src/main/webapp/newsfeed.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  
+    return update(ref(database), updates);
+}
+
+// retrieve a list of posts
+export function getPosts() {
+    const dbRef = ref(database);
+    get(child(dbRef, `posts/`)).then((snapshot) => {
+    if (snapshot.exists()) {
+        console.log(snapshot.val());
```
Comment (sarahrittgers26) [inline src/main/webapp/newsfeed.js:34]: Maybe add a TODO comment to remove these console.log statements once we're actually reading the response?

File: src/main/webapp/newsfeed.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    } else {
+        console.log("No data available");
+    }
+    }).catch((error) => {
+    console.error(error);
```
Comment (sarahrittgers26) [inline src/main/webapp/newsfeed.js:39]: Please indent this line

**Review by sarahrittgers26 — APPROVED** [review]
Thanks!

**Review by pabloalmada200 — APPROVED** [review]

**Review by SimGon-lab — APPROVED** [review]
great job on comments too

**Pull request merged**